content/800-data-platform/100-accelerate/200-getting-started.mdx

```diff
@@ -82,26 +90,22 @@ Run the following command to install the Accelerate extension for Prisma Client:
 npm install @prisma/extension-accelerate
 ```
 
-If your Prisma version is below `4.16.1`, enable the Prisma Client extension feature:
-
-```prisma
-/// prisma/schema.prisma
-generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["clientExtensions"]
-}
-```
-
 ### 2.3. Generate Prisma Client for Accelerate
 
 Run the following command to generate Prisma Client for Accelerate:
 
 ```terminal
-npx prisma generate --accelerate
+npx prisma generate --no-engine
```

Comment:
This is only necessary if you want to strip out the engines, right? It's recommended, but not required? Is that right, and if so, should we mention that?

Reply:
@petradonka okay, I'm explicitly mentioning that 🙏

Reply:
Please only do if what I'm saying is actually correct 😄

Reply:
@petradonka - you are correct 😄 . I skipped the explanation initially because the section was as lengthy as it is. Here's the [reference](https://github.com/prisma/prisma/releases/tag/5.2.0).

I've updated the section in the latest commit.